latch_cli/config/user.py

```diff
@@ -64,9 +86,9 @@ def update_token(self, token: str):
         with open(self.token_path, "w") as f:
             f.write(token)
 
-    def update_workspace(self, workspace: str):
+    def update_workspace(self, workspace_id: str, name: Optional[str] = None):
         with open(self.workspace_path, "w") as f:
-            f.write(workspace)
+            f.write(json.dumps({"workspace_id": workspace_id, "name": name}))
```

Comment:
weird bc ws name can change.

Reply:
yeah just include the name in your node data query instead?

Reply:
true, but i dont think its that big a deal, given switching workspaces will refresh it anyway